packages/story-editor/src/components/form/color/color.js

```diff
@@ -56,7 +56,7 @@ const Container = styled.section`
 `;
 Container.propTypes = {
   isInDesignMenu: PropTypes.bool,
-  width: PropTypes.string,
+  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
this is just silencing a console warning pre release while i'm here.

Reply:
Is this because we're now passing an empty string to `width`? Can't we pass `null` instead?

Reply:
Not sure why it was `string` before, but changing that to `width: PropTypes.number,` seems to work for all cases.